src/main/java/org/scijava/io/AbstractDataHandle.java

```diff
@@ -71,4 +77,23 @@ public void setEncoding(final String encoding) {
 		this.encoding = encoding;
 	}
 
+	@Override
+	public void writeUTF(String str) throws IOException {
```

Comment:
This is not a new default method of `DataHandle`. I.e.: this change does not match the commit message. It should be done separately.